setup.cfg

```diff
@@ -24,7 +24,7 @@ classifiers =
 [options]
 include_package_data = true
 packages = find:
-python_requires = >= 3.10
+python_requires = >= 3.9
```

Comment:
Unless you changed things to be backward compatible, other sections of code will not work with 3.9 due to syntax that was introduced in 3.10

Reply:
I've updated to 3.10 by making our serverless worker compatible with 3.10 now.